var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -29,19 +29,19 @@
 from spack import *
 
 
-class Rose(Package):
+class Rose(AutotoolsPackage):
     """A compiler infrastructure to build source-to-source program
        transformation and analysis tools.
        (Developed at Lawrence Livermore National Lab)"""
 
     homepage = "http://rosecompiler.org/"
-    url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
+    #url = "https://github.com/rose-compiler/rose/archive/v0.9.7.tar.gz"
```

Comment:
> We don't support the GIthub tarballs.

What do you mean by this? I see you have a `0.9.7.0` version. Maybe you could change it to this? Checking out a specific commit is nice, but it makes it difficult to manage a mirror for users on air-gapped clusters.